core/src/Cache/MsalCacheCommon.cs

```diff
@@ -38,9 +38,9 @@ internal class MsalCacheCommon
         public const string ScopesDelimiter = " ";
         public const string CacheKeyDelimiter = "-";
 
-        public const string IdToken = "idtoken";
-        public const string AccessToken = "accesstoken";
-        public const string RefreshToken = "refreshtoken";
+        public const string IdToken = "IdToken";
+        public const string AccessToken = "AccessToken";
+        public const string RefreshToken = "RefreshToken";
```

Comment:
not sure I understand why you are CamelCasing these. Though I see that the methods lowercases everything.

Reply:
According to current schema design doc https://identitydivision.visualstudio.com/DevEx/DevEx%20Team/_git/AuthLibrariesApiReview?path=%2FUnifiedSchema%2FSchema.md&version=GBoldalton%2Funified_schema_transfer
credential_type is case sensitive, but all keys are lowercased

Reply:
ok... not sure though why the values here are CamelCased (if it's being lower cased later anyway).

---
In reply to: [227980008](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350#discussion_r227980008) [](ancestors = 227980008)